src/pages/index.js

```diff
@@ -44,29 +44,34 @@ let cardSection;
 // functions
 function handleProfileFormSubmit(data) {
   const { name, about } = data;
+  profileEditPopup.editSubmitBtn();
   api
     .updateProfile(name, about)
-    .then(profileEditPopup.editSubmitBtn())
     .then(() => {
       userInfo.setUserInfo(name, about);
     })
+    .then(() => {
+      profileEditPopup.close();
+    })
     .catch((err) => {
       console.error(err);
     })
     .finally(() => {
       profileEditPopup.resetSubmitBtn();
-      profileEditPopup.close();
     });
 }
 
 function handleCardFormSubmit(data) {
+  profileAddPopup.editSubmitBtn();
   api
     .postCard(data)
-    .then(profileAddPopup.editSubmitBtn())
     .then((data) => {
       const card = createCard(data);
       cardSection.prependItem(card);
     })
+    .then(() => {
+      profileAddPopup.close();
+    })
     .catch((err) => {
       console.error(err);
     })
```

Comment:
`profileAddPopup.close();` should be deleted from line 80. You need to close only in `then`


```js
function handleCardFormSubmit(data) {
  profileAddPopup.editSubmitBtn();
  api
    .postCard(data)
    .then((data) => {
      const card = createCard(data);
      cardSection.prependItem(card);
    })
    .then(() => {
      profileAddPopup.close();
    })
    .catch((err) => {
      console.error(err);
    })
    .finally(() => {
      profileAddPopup.resetSubmitBtn();
    });
}

```